### PR TITLE
Add a configuration reference document for helm chart

### PR DESCRIPTION
**Pull request by dmolenda-sumo**
- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request adds a configuration reference document to the Sumo docs

OSC-502

## Select the type of change:
- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

**Comment by jpipkin1** [comment]
@kimsauce, I'll leave the review to you since you're our Helm Chart writer.

File: docs/send-data/kubernetes/configuration.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+id: configuration
+title: Configuration
+sidebar_label: Configuration
+description: Sumo Logic Kubernetes Helm Chart configuration.
+---
+
+Detailed list of Sumo Logic Kubernetes Helm Chart configurable parameters are available in [project repository](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/deploy/helm/sumologic/README.md)
```
Comment (kimsauce) [inline docs/send-data/kubernetes/configuration.md:8]: @dmolenda-sumo do you plan to migrate [this page](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/deploy/helm/sumologic/README.md) to our Docs or will it be staying there?

**Review by kimsauce — COMMENTED** [review]
Question

File: docs/send-data/kubernetes/configuration.md (hunk 1 above)
Comment (dmolenda-sumo) [reply docs/send-data/kubernetes/configuration.md:8]: We have no plans to move this document. There is a script checks whether all fields in the config are explained, and we want to keep this feature.

**Review by kimsauce — APPROVED** [review]

File: docs/send-data/kubernetes/configuration.md (hunk 1 above)
Comment (kimsauce) [inline docs/send-data/kubernetes/configuration.md:8]:
```suggestion
See our [Sumo Logic Kubernetes Collection project repository](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/deploy/helm/sumologic/README.md) for a complete list of configurable Helm Chart parameters.
```

**Review by kimsauce — APPROVED** [review]

**Pull request merged**